plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/GenerateLibrary.kt

```diff
@@ -0,0 +1,66 @@
+package godot.gradle
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+import org.gradle.kotlin.dsl.mapProperty
+import org.gradle.kotlin.dsl.property
+import java.io.File
+
+open class GenerateLibrary : DefaultTask() {
```

Comment:
I'm more for `GenerateGdns`. `GenerateLibrary` sounds like you actually generate a whole lib, which is not the case